Examples/fortran/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+##---------------------------------------------------------------------------##
```

Comment:
I'd like CMake support in swig-4.0, but I'd like it done consistently and it'll probably be based off the pull requests #1039 #1040.  I want to focus on Fortran for now and not get distracted by CMake holding up this Fortran pull request.  Fortran CMake support will need to fit in with the overall CMake support when added and there are fundamental details about the CMake vision that are not clear to me.

So for now, please remove these files from this pull request:

Examples/fortran/CMakeLists.txt
Examples/fortran/arrayview/CMakeLists.txt
Examples/fortran/barefunctions/CMakeLists.txt
Examples/fortran/bindc/CMakeLists.txt
Examples/fortran/class/CMakeLists.txt
Examples/fortran/cmake/.gitignore
Examples/fortran/cmake/SwigModules.cmake
Examples/fortran/cmake/config.example.cmake
Examples/fortran/cmake/old_cmake/UseSWIG.cmake
Examples/fortran/except/CMakeLists.txt
Examples/fortran/funcptr/CMakeLists.txt
Examples/fortran/inheritance/CMakeLists.txt
Examples/fortran/mpitest/CMakeLists.txt
Examples/fortran/simple/CMakeLists.txt
Examples/fortran/simpleclass/CMakeLists.txt
Examples/fortran/simplest/CMakeLists.txt
Examples/fortran/spdemo/CMakeLists.txt
Examples/fortran/static_members/CMakeLists.txt
Examples/fortran/std_string/CMakeLists.txt
Examples/fortran/std_vector/CMakeLists.txt
Examples/fortran/templated/CMakeLists.txt
Examples/fortran/thinvec/CMakeLists.txt